baracoda/child_barcodes.py

```diff
@@ -0,0 +1,66 @@
+import logging
```

Comment:
Wondering if a bit more documentation would be nice? Just to explain why this is here and what it is doing.

Reply:
Ok, added more documentation